### DISCUSSION TITLE
Android users, listen up! - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#628] [Announcements]
# Android users, listen up! - Revolt

One year in the making, here it comes.

https://revolt.chat/posts/revolt-on-the-go

**Comment by tirednessa** [comment]
10/10 app can confirm

**Comment by Inderix** [comment]
top ten epic moments of revolt history

**Comment by Rexogamer** [comment]
as i have a conflict of interest i will not comment further :trollface:

**Comment by 1enify** [comment]
been waiting for so long, hope it will work on my phone running android 10

**Reply by Rexogamer** [reply]
The minimum version will be Android 7 (SDK 24 for you fellow :nerd_face:s) so yes, your phone will be supported ^^

**Comment by QueenOworld** [comment]
Real

**Comment by taskylizard** [comment]
so true bestie

**Comment by retcinder** [comment]
:3

**Comment by NigelL12** [comment]
👀

**Comment by aeongdesu** [comment]
I'm going to explode

**Comment by clot27** [comment]
common W

**Comment by searinminecraft** [comment]
material you support???

(there probably will be as i saw mention of it in translations)

**Reply by cyrneko** [reply]
what you're seeing in the screenshots already is Material You :trollface:

**Comment by ghost** [comment]
epic :trollface:

**Comment by blitzdose** [comment]
Really amazing work. Now it would be even better if the self hosting would work flawlessly and then I could finally end my relationship with discord

**Comment by FanMclaine** [comment]
common revolt W

**Comment by 1iR00Kod3r** [comment]
Looking forward to this :D!! I have Android 13 so I'm not sure if that's possible.

**Reply by Rexogamer** [reply]
Android 13 will be supported ^^

**Reply by 1iR00Kod3r** [reply]
Ooo! Ty for the heads up! ^^/

**Comment by Niterux** [comment]
I really hope this supports android tablet, the current app is a PAIN to use on android tablet because it force landscape

**Reply by Niterux** [reply]
i meant to say portrait not landscape but this ui is confusing and i do not know how to edit my response, sorry

**Reply by searinminecraft** [reply]
![Screenshot_20231109-193907](https://github.com/revoltchat/revolt/assets/114207889/29ee9ebe-3edf-47ea-94ea-ee904c66b120)

**Reply by Niterux** [reply]
I don't have that button that ui is completely different to what i see on my screen :(

**Reply by Rexogamer** [reply]
It looks like you can't edit comments via Giscus itself - you'll have to go to [the original GitHub discussion](https://github.com/orgs/revoltchat/discussions/628) instead

**Reply by Kubrix** [reply]
Why does not *.apk for this app avaialble on GIthub?
WHy do I need google play ON and last version for this app to work?  

Like discord does not need that

**Reply by 1enify** [reply]
> WHy do I need google play ON and last version for this app to work?
> 
> Like discord does not need that

To my understanding it was not a decision made by Jennifer (the developer). It may be because of material you, but I doubt it, it might be Google's shenanigans instead. You can download app v1.3.3, it's the latest one that doesn't need to be exclusively downloaded from Google Play. Or you can add [Revolt's f-droid repo](https://fdroid.revolt.chat/repo/?fingerprint=0a9d2f61c8659801711e22177862f84c8134966f427973037a8fdfacff07c4f2) to your client and use the pwa.

**Reply by Kubrix** [reply]
> You can download app v1.3.3,
Unfortunatelly  I can find only play store link on https://revolt.chat/download/android and no direct links in https://github.com/revoltchat/android

Why I need F-Droid client? Why cant site just host *.apk?
If you want to be Discord alternative then give user a way to get app without needing  google play or just give *.apk

I am not installing whole android studio just to build app

**Reply by Niterux** [reply]
What are you talking about, Discord only distributes their app on Google Play and the App Store 🤣. This is such a silly and meaningless nitpick, every Android phone comes with Google Play installed. You don't even need the F-Droid client to download apps from F-Droid you can just download it from the F-Droid website.

**Comment by Kubrix** [comment]
Sure pal:
https://search.f-droid.org/?q=Revolt
**It looks like F-Droid does not have any apps matching your search string "Revolt"**

**Reply by 1enify** [reply]
> Sure pal: https://search.f-droid.org/?q=Revolt **It looks like F-Droid does not have any apps matching your search string "Revolt"**

No, you need to add the repo, then search for Revolt. Copy the link I embedded and paste it into f-droid repositories(in settings). I'm sure there are plenty of videos going in depth about f-droid repositories on youtube.

**Comment by Kubrix** [comment]
IF you want people to use revolt you have to make it easier to download and use it

Discord does not need Google play to work

Mullvad vpn can be downloaded from F-droid site. No need to get via client
